client/my-sites/upgrades/map-domain/index.jsx

```diff
@@ -1,26 +1,29 @@
 /**
  * External dependencies
  */
-var page = require( 'page' ),
-	React = require( 'react' ),
-	omit = require( 'lodash/omit' ),
-	{ connect } = require( 'react-redux' );
+import page from 'page';
+import React from 'react';
+import omit from 'lodash/omit';
```

Comment:
Might be a good idea to also convert such instances to use root Lodash imports, for example:

```
import { omit } from 'lodash';
```

See #6539 for more information.